### PR TITLE
Minor fixes new skills

### PR DESCRIPTION
**Pull request by gemfarmer**
@detre I made this into a Pull Request so that it will be easier to track changes.

I made several suggestions. I realize I sound harsh, but really great job. Just trying to be to the point with my suggestions.

### CODE REVIEW

File: client/app/main/main.scss [hunk 1]
```diff
@@ -79,18 +79,40 @@
 }
 
 .community-image {
-    width: 90%;
+    width: 180px;
     float: left;
-    margin-right: $base-padding;
+    margin-right: 20px;
```
Comment (gemfarmer) [inline client/app/main/main.scss]: Padding should never be pixels (for responsiveness), and this should be stored as a variable. `$`

File: client/app/main/main.scss [hunk 2]
```diff
@@ -79,18 +79,40 @@
...
     margin-left: $base-padding;
 }
-.brainstorm-image {
-    width: 90%;
-    float: left;
-    margin-right: $base-padding;
+.world-image {
+    width: 140px;
+    float: right;
+    margin-right: 20px;
```
Comment (gemfarmer) [inline client/app/main/main.scss]: See above ^^

File: client/app/main/warmupModal.html [hunk 3]
```diff
@@ -1,10 +1,61 @@
...
+<html>
+    <head>
+        <title>warmup</title>
+
 <!--<div>Pre-dream warm-up text here.</div>-->
-<img class="community-image" src="assets/images/community.png"/>
-<!--<div>Dream Brainstorm
-Take 5-8 minutes to put your dreams and goals on paper.  Feel free to brainstorm with lists, drawings, or figures etc., whatever you need to get the ideas flowing. This will be yours to keep and private to you. Your dreams may look very different today then tomorrow and that is okay. You’re just trying to get the ideas flowing and on paper.
+</head>
+     <div class="texty">
```
Comment (gemfarmer) [inline client/app/main/warmupModal.html]: please use a more semantic class name

File: client/app/main/warmupModal.html [hunk 4]
```diff
@@ -1,10 +1,61 @@
...
+ 	<h3 id="title2"> Affirm and Inspire </h3>
+    <body id="textBody">
```
Comment (gemfarmer) [inline client/app/main/warmupModal.html]: you probably meant to use a `<section>` tag here? Also, make sure to close all tags or the site will start to look buggy. `<section></section>`

File: client/app/main/warmupModal.html [hunk 5]
```diff
@@ -1,10 +1,61 @@
...
+        </slide>
+      </carousel>
+    </div>
+  </div>
+</div>  
+<script type="text/javascript" src="https://ajax.googleapis.com/ajax/libs/angularjs/1.0.8/angular.min.js"></script>
+	<script type="text/javascript" src="//cdnjs.cloudflare.com/ajax/libs/angular-ui-bootstrap/0.10.0/ui-bootstrap-tpls.min.js"></script> -->
+
+     <div class="texty">
+ 	<h3 id="title2"> Values, Community and Shared Goals</h3>
```
Comment (gemfarmer) [inline client/app/main/warmupModal.html:30]: use a class instead of an id and these h3s can have shared styles

File: client/app/routes/resources/resources.html [hunk 6]
```diff
@@ -1,11 +1,51 @@
 <div ng-include="'components/navbar/navbar.html'"></div>
 <div ng-include="'components/sideNav/sideNav.html'" ></div>
-
+<head>
```
Comment (gemfarmer) [inline client/app/routes/resources/resources.html]: there should be no additional `<head>` tag

File: client/app/routes/resources/resources.html [hunk 7]
```diff
@@ -1,11 +1,51 @@
 <div ng-include="'components/navbar/navbar.html'"></div>
 <div ng-include="'components/sideNav/sideNav.html'" ></div>
-
+<head> 
+<base target="_blank">
```
Comment (gemfarmer) [inline client/app/routes/resources/resources.html:4]: what is `<base>`??

File: client/app/routes/resources/resources.html [hunk 8]
```diff
@@ -1,11 +1,51 @@
...
+</head>
 <div class="page">
 	<section class="container">
 
-		This is the resources view. More to come!
+		Here are some resources. More to come!
+ <img class="resources-image" src="assets/images/resources7text.png"/>
+ <div class="vision-board">
```
Comment (gemfarmer) [inline client/app/routes/resources/resources.html:13]: None of these `<div>` tags have ending tags, and should be `<ul>` tags if you are going to but `<li>` tags in them

File: client/app/routes/resources/resources.scss [hunk 9]
```diff
@@ -0,0 +1,32 @@
...
+  	margin-top: $base-padding;
+    margin-right: $base-padding;
+    margin-left: $base-padding;
+}
+
+.vision-board {
+    top: 80px;
+    left: 355px; 
+}
+.goal-setting {
```
Comment (gemfarmer) [inline client/app/routes/resources/resources.scss:13]: It would be more elegant/readable/maintainable if all of these settings could shared a common class.

File: client/components/sideNav/sideNav.scss [hunk 10]
```diff
@@ -49,7 +49,7 @@
 		text-transform: uppercase;
 		&:hover,
 		&:focus {
-			color: $navy;
+			color: #b3ccff;
```
Comment (gemfarmer) [inline client/components/sideNav/sideNav.scss]: please make a variable for this.

File: client/app/main/warmupModal.html (hunk 3 above)
Comment (detre) [inline client/app/main/warmupModal.html]: whaaat! texty is the best!

File: client/app/routes/resources/resources.html (hunk 7 above)
Comment (detre) [inline client/app/routes/resources/resources.html:4]: not sure.  this just opens up the link in a new tab.

**Comment by detre** [comment]
Hi Brian, I just updated my branch.  Starting from the top: I changed all the px to % but didn't necessarily change to $, I closed all my tags (sorry) or removed them if unnecessary.  I am still not sure what, when or why you use id, div, section etc. but tried to follow the order I found when researching about them. I  didn't follow your guidance for a couple of the edits, as it would change how it looked. But I did make it a bit more responsive to the mobile settings.  Let me know if the <ul> before the <li> is a hard fast rule.  It indents my list which I do not want. I also didn't change the resources.scss as my original plan was to move those sections around so its not a list.  Thanks, so much for helping me in my learning process.  I am rarely a newbie anymore. It builds compassion for my students.  We should chat about next steps soon.  

Detre

**Comment by detre** [comment]
haha that bullet point is not supposed to be there. It should be " let me know if the ul before the li is a hard fast rule".
